annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -0,0 +1,6 @@
+spring.batch.job.enabled=false
+genomenexus.base_url=
+genomenexus.hgvs=variant_annotation/hgvs/
+genomenexus.isoform_override=variant_annotation/isoform_override/
```

Comment:
I think we should make this switch.  This way we can use the same compiled codebase in our cronjob to annotate the MSK-IMPACT data (MSK isoforms) for the MSK portal or public studies (UniProt) for the public portal.